src/main/java/com/javadiscord/javabot/Bot.java

```diff
@@ -55,6 +59,11 @@
      */
     public static HikariDataSource dataSource;
 
+    /**
+     * A list of known links used for scamming. (from https://github.com/DevSpen/links)
+     */
+    public static List<String> suspiciousLinks;
```

Comment:
I don't think that this should be done here, in the bot's main class, because these suspicious links aren't something that will be used by tons of different components. It would be better if you just loaded them in the AutoMod's constructor or something.

Reply:
Yeah, I'll fix that later today. Dynxsty also said, that it won't work because if compiled to a jar there's no src folder so I need to fix that as well